data/mods/Magiclysm/Spells/magus.json

```
@@ -42,7 +42,32 @@
     "spell_class": "MAGUS",
     "base_casting_time": 100,
     "energy_source": "MANA",
-    "base_energy_cost": 100
+    "base_energy_cost": 100,
+    "learn_spells": { "dimension_door": 10 }
+  },
+  {
+    "id": "dimension_door",
+    "type": "SPELL",
+    "name": "Dimension Door",
+    "description": "Teleports you randomly near a target location.",
+    "effect": "teleport_to",
+    "shape": "blast",
+    "valid_targets": [ "ground" ],
+    "flags": [ "SOMATIC", "SILENT", "NO_LEGS" ],
+    "min_range": 10,
+    "max_range": 20,
+    "range_increment": 1,
+    "min_aoe": 6,
+    "max_aoe": 3,
+    "aoe_increment": -0.3,
+    "max_level": 10,
```

```suggestion
    "max_level": 30,
```
this spell is ok, i just want the stats of it stretched out more. getting to level 10 in a spell is fairly trivial, but 20 or 30 is not. this spell is a lot more powerful than the other teleport spells so i want there to be some kind of "downside" - in this case, a bit slow to cast and needing extra time to study.